docs/DeveloperGuide.md

```diff
@@ -562,7 +562,7 @@ Similar to [<ins>UC01](#use-case-uc01-list-all-contacts) except to view tasks in
 Similar to [<ins>UC04](#use-case-uc04-create-a-contact) except adding a wedding to WedLinker instead of a Contact
 
 ### **Use case: UC06 — Create a Task**
-Similar to [<ins>UC04](#use-case-uc04-create-a-contact) except adding a task to WedLinker instead of a Contact
+Similar to [<ins>UC04]( #use-case-uc04-create-a-contact) except adding a task to WedLinker instead of a Contact
```

Comment:
Just wondering, is there a particular reason for the added space, or maybe it was an unintentional addition

Reply:
yep unintentional, thanks for pointing out!